service/history/shard/controller_impl.go

```diff
@@ -132,6 +135,7 @@ func (c *ControllerImpl) Start() {
 	}
 
 	c.ownership.start(c)
+	c.dlqMetricsEmitter.Start()
```

Comment:
hmmm don't think it belong there. Maybe move it to history handler?

Reply:
Moved.